packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -58,7 +59,22 @@ const DefaultChunkSize = 16 * 1024;
 const ImmediateNoOpResponse = "";
 
 // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-const getRetryDelayFromError = (error: any): number | undefined => error?.retryAfterSeconds;
+export const getRetryDelayFromError = (error: any): number | undefined => error?.retryAfterSeconds;
+
+export function emitThrottlingWarning(
+    delayTime: number,
+    error: ICriticalContainerError,
+    emitter: Container | DeltaManager,
+) {
```

Comment:
Also I believe that throttling warning should be raised on container in all cases. For delta manager we raise it on delta manager, I think we should change that to raise it on container.

Reply:
We already do - here is code in Container.ts:
```
        deltaManager.on("throttled", (warning: IThrottlingWarning) => {
            this.raiseContainerWarning(warning);
        });
```